pkg/pillar/attest/attest.go

```diff
@@ -203,6 +205,7 @@ var transitions = map[Transition]EventHandler{
 
 //some helpers
 func triggerSelfEvent(ctx *Context, event Event) error {
+	log.Infof("Creating %s at %s", "func", agentlog.GetMyStack())
```

Comment:
Hi @eriknordmark is this really required? Or are we adding this since ctx can be null, and we want to catch crash at line 207?

Reply:
I'll drop that one.